DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Helpers/GraphSyncHelper.cs

```diff
@@ -139,8 +143,8 @@ public async Task<string> GenerateIdPropertyValue()
                 "$\"http://data.europa.eu/esco/{ContentType.ToLowerInvariant()}/{Value}\"" =>
                 $"http://data.europa.eu/esco/{_contentType!.ToLowerInvariant()}/{newGuid}",
 
-                "$\"http://nationalcareers.service.gov.uk/{ContentType.ToLowerInvariant()}/{Value}\"" =>
-                $"http://nationalcareers.service.gov.uk/{_contentType!.ToLowerInvariant()}/{newGuid}",
+                "$\"<<ContentApiPrefix>>/{ContentType.ToLowerInvariant()}/{Value}\"" =>
+                $"{_contentApiBaseUrl}/{_contentType!.ToLowerInvariant()}/{newGuid}",
 
                 _ => await TransformOrDefault(GraphSyncPartSettings!.GenerateIdPropertyValue, newGuid, _contentType!)
```

Comment:
I think it'd be worthwhile to imbue the general (non-optimized) case with the ability to inject ContentApiPrefix.